bindings/core/src/method/wallet_command.rs

```diff
@@ -18,28 +18,18 @@ use iota_sdk::{
         payload::signed_transaction::TransactionId,
     },
     wallet::{
-        account::{
-            ConsolidationParams, CreateAccountParams, CreateNativeTokenParams, FilterOptions, MintNftParams,
-            OutputParams, OutputsToClaim, SyncOptions, TransactionOptions,
-        },
-        SendNativeTokensParams, SendNftParams, SendParams,
+        ConsolidationParams, CreateAccountParams, CreateNativeTokenParams, FilterOptions, MintNftParams, OutputParams,
+        OutputsToClaim, SendNativeTokensParams, SendNftParams, SendParams, SyncOptions, TransactionOptions,
     },
     U256,
 };
 use serde::{Deserialize, Serialize};
 
-/// Each public account method.
+/// Each public wallet command method.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 #[non_exhaustive]
-pub enum AccountMethod {
-    /// List addresses.
-    /// Expected response: [`Addresses`](crate::Response::Addresses)
-    Addresses,
-    /// Returns only addresses of the account with unspent outputs
-    /// Expected response:
-    /// [`AddressesWithUnspentOutputs`](crate::Response::AddressesWithUnspentOutputs)
-    AddressesWithUnspentOutputs,
+pub enum WalletCommandMethod {
```

Comment:
I don't really like this split/name, will put in my issue to not have to address it here.

Reply:
you mean ... just like Chloe you want to merge everything into `WalletMethod`?

Reply:
I think that would be my preference yes, there is no such dichotomy in the SDK anymore so I don't know why there would be one in the bindings. At the very least we rename the enum, `WalletCommandMethod` is way too weird. But just keep it like that for now, we can change later.

Reply:
Done :trollface: